kubetest/kops.go

```diff
@@ -36,9 +36,8 @@ import (
 	"time"
 
 	"cloud.google.com/go/storage"
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
```

Comment:
are we even using kubetest/kops anymore?

(considering that should be a follow-up)

I'm pretty sure kops has fully adopted kubetest2 for their own testing.